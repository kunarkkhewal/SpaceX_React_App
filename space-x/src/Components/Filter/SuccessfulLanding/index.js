import React from 'react';
import BooleanButton from './../BooleanButton';
import './../index.css'

const SuccessfulLanding = () => {
    return(
        <div className="SuccessfulLanding-div">
            <h2>Successful Landing</h2>
            {<BooleanButton value='land_success'/>}
        </div>
    )
}

export default SuccessfulLanding;