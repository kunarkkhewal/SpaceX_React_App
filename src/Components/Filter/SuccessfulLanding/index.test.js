import React from "react";
import { mount } from "enzyme";
import SuccessfulLanding from "./index";

describe("SuccessfulLanding", () => {
    it('should render SuccessfulLanding component correctly', () => {
        const component = mount(<SuccessfulLanding/>);

        expect(component).toMatchSnapshot();
    });
});