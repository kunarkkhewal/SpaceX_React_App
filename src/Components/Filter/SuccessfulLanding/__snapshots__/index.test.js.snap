// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SuccessfulLanding should render SuccessfulLanding component correctly 1`] = `
<SuccessfulLanding>
  <div
    className="SuccessfulLanding-div"
  >
    <h2>
      Successful Landing
    </h2>
    <BooleanButton
      value="land_success"
    >
      <div
        className="boolean-button-div"
      >
        <button
          onClick={[Function]}
          value="true"
        >
          TRUE
        </button>
        <button
          onClick={[Function]}
          value="false"
        >
          FALSE
        </button>
      </div>
    </BooleanButton>
  </div>
</SuccessfulLanding>
`;
